[ANGLE] Build broken when targeting Android

aperezdc opened the pull request:
#### 67af852f97b2994d601132f9f2cbeec7b78261b1
<pre>
[ANGLE] Build broken when targeting Android
<a href="https://bugs.webkit.org/show_bug.cgi?id=245093">https://bugs.webkit.org/show_bug.cgi?id=245093</a>

Reviewed by Kimmo Kinnunen.

* Source/ThirdParty/ANGLE/CMakeLists.txt: Choose the appropriate
  translator symbol table sources on Android.

Canonical link: <a href="https://commits.webkit.org/254426@main">https://commits.webkit.org/254426@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8ce8e7fb4b13c44d3b1de6c637801d6d466b9a67

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/88970 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/33533 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/19842 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/98251 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/154632 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/32038 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/27631 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/81333 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/92785 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/94600 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/68/builds/25443 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/75942 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/25378 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/80310 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/80327 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/68345 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/29829 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/14372 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/29560 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/15352 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/3101 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/32999 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/38295 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/31689 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/34451 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

kkinnunen-apple commented on Source/ThirdParty/ANGLE/CMakeLists.txt:
```diff
@@ -77,14 +77,19 @@ set(ANGLE_SOURCES
     ${angle_translator_glsl_and_vulkan_base_sources}
     ${angle_translator_glsl_sources}
-    ${angle_translator_glsl_symbol_table_sources}
     ${angle_translator_sources}
```
I think it's in fact correct to use the essl symbol table unilaterally.
This is the frontend symbol table. The ANGLE frontend for WebGL is always GL ES.
(it shouldn't be a problem using the glsl, it's just overhead.)

aperezdc replied:
I went with this fix to cause as little disruption as possible, but that's a good point so I have filed [bug #245126](https://bugs.webkit.org/show_bug.cgi?id=245126) to handle that change later on.

aperezdc committed:
[ANGLE] Build broken when targeting Android
https://bugs.webkit.org/show_bug.cgi?id=245093

Reviewed by Kimmo Kinnunen.

* Source/ThirdParty/ANGLE/CMakeLists.txt: Choose the appropriate
  translator symbol table sources on Android.

Canonical link: https://commits.webkit.org/254426@main
